keadm/cmd/keadm/app/cmd/debug/get.go

```diff
@@ -0,0 +1,901 @@
+keadm debug get all -o yaml`
+
+	// availableResources Convert flag to currently supports available Resource types in EdgeCore database.
+	availableResources = map[string]string{
+		"all":        ResourceTypeAll,
+		"po":         model.ResourceTypePod,
+		"pod":        model.ResourceTypePod,
+		"pods":       model.ResourceTypePod,
+		"no":         model.ResourceTypeNode,
+		"node":       model.ResourceTypeNode,
+		"nodes":      model.ResourceTypeNode,
+		"svc":        constants.ResourceTypeService,
+		"service":    constants.ResourceTypeService,
+		"services":   constants.ResourceTypeService,
+		"secret":     model.ResourceTypeSecret,
+		"secrets":    model.ResourceTypeSecret,
+		"cm":         model.ResourceTypeConfigmap,
+		"configmap":  model.ResourceTypeConfigmap,
+		"configmaps": model.ResourceTypeConfigmap,
+		"ep":         constants.ResourceTypeEndpoints,
+		"endpoint":   constants.ResourceTypeEndpoints,
+		"endpoints":  constants.ResourceTypeEndpoints,
+	}
+)
+
+// NewCmdDebugGet returns keadm debug get command.
+func NewCmdDebugGet(out io.Writer, getOption *GetOptions) *cobra.Command {
+	if getOption == nil {
+		getOption = NewGetOptions()
+	}
+
+	cmd := &cobra.Command{
+		Use:     "get",
+		Short:   "Display one or many resources",
+		Long:    debugGetLong,
+		Example: debugGetExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := getOption.Validate(args); err != nil {
+				CheckErr(err, fatal)
+			}
+			if err := getOption.Run(args, out); err != nil {
+				CheckErr(err, fatal)
+			}
+		},
+	}
+	addGetOtherFlags(cmd, getOption)
+
+	return cmd
+}
+
+// fatal prints the message if set and then exits.
+func fatal(msg string, code int) {
+	if len(msg) > 0 {
+		// add newline if needed
+		if !strings.HasSuffix(msg, "\n") {
+			msg += "\n"
+		}
+
+		fmt.Fprint(os.Stderr, msg)
+	}
+	os.Exit(code)
+}
+
+// CheckErr formats a given error as a string and calls the passed handleErr
+// func with that string and an exit code.
+func CheckErr(err error, handleErr func(string, int)) {
+	switch err.(type) {
+	case nil:
+		return
+	default:
+		handleErr(err.Error(), DefaultErrorExitCode)
+	}
+}
+
+// addGetOtherFlags
+func addGetOtherFlags(cmd *cobra.Command, getOption *GetOptions) {
+	cmd.Flags().StringVarP(&getOption.Namespace, "namespace", "n", getOption.Namespace, "List the requested object(s) in specified namespaces")
+	cmd.Flags().StringVarP(getOption.PrintFlags.OutputFormat, "output", "o", *getOption.PrintFlags.OutputFormat, "Indicate the output format. Currently supports formats such as yaml|json|wide")
+	cmd.Flags().StringVarP(&getOption.LabelSelector, "selector", "l", getOption.LabelSelector, "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
+	cmd.Flags().StringVarP(&getOption.DataPath, "input", "i", getOption.DataPath, "Indicate the edge node database path, the default path is \"/var/lib/kubeedge/edgecore.db\"")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&getOption.DataPath, "edgedb-path", "p", getOption.DataPath, "Indicate the edge node database path, the default path is \"/var/lib/kubeedge/edgecore.db\"")
```
How about use `edgedb-path` instead of `input`?  @qingchen1203 @shenkonghui @GsssC

Reply:
agree : )

Reply:
Okey, please fix this comment and make the ci passed. And please squash the commits a bit less. Then we can get it merged first.